application/lightmeter.h

```diff
@@ -19,14 +22,19 @@
 #include "lightmeter_config.h"
 #include <BH1750.h>
 
+#define APP_PATH_DIR "/ext/lightmeter"
```

Comment:
According to the convention, app data should be stored in /ext/apps_data/lightmeter